README.md

```diff
@@ -1415,6 +1415,26 @@
     };
 
     return obj;
+
+    // bad
+    const obj = [
```

Comment:
maybe `arr` would be a better reference name here? I thought this was a typo for a moment -_-